src/OpenTelemetry/Metrics/DataPoint{T}.cs

```diff
@@ -0,0 +1,46 @@
+// <copyright file="DataPoint{T}.cs" company="OpenTelemetry Authors">
```

Comment:
is that a usual practice to have { in file names?

Reply:
Yes, this was very strange to me as well. I googled for this answer.

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1730